### ISSUE TITLE
Duplicated postfix completion suggestions

### ISSUE THREAD

**Issue by boobiq** [xylo/intellij-postfix-templates#144]
Hi, all postfix completion suggestions from plugin are tripled for some reason. I tried to reinstall plugin, invalidated cache and restarted GoLand. Not sure when it started to happen, I'm not Nightly versions of GoLand so maybe that could "help".

![image](https://user-images.githubusercontent.com/429225/65745499-352c8000-e0fc-11e9-9633-97d3ab4f4b35.png)

**Comment by xylo** [comment]
I could reproduce the bug on my system:
```
GoLand 2019.2.2
Build #GO-192.6603.35, built on September 11, 2019
Runtime version: 11.0.3+12-b304.56 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 5.0.0-29-generic
GC: ParNew, ConcurrentMarkSweep
Memory: 1979M
Cores: 12
Registry: run.processes.with.pty=TRUE, ide.completion.variant.limit=500, suggest.all.run.configurations.from.context=true
Non-Bundled Plugins: de.endrullis.idea.postfixtemplates
```

**Comment by xylo** [comment]
But it looks to be fixed in Goland 2020.1.3.
